scripts/nolints.json

```diff
@@ -1,46 +1 @@
-[["docBlame", "Std.BitVec.reduceAbs"],
```

Comment:
I wouldn't have expected to see this changes: is this a bug or were these entries stale before?

Reply:
the entries were stale. (I think they were all upstreamed before fixing though, and core doesn't track docBlame because it has a huge number of violators to begin with. :( )